crates/uv/tests/it/python_find.rs

```diff
@@ -28,7 +28,8 @@ fn python_find() {
         ----- stdout -----
 
         ----- stderr -----
-        error: No interpreter found in virtual environments, managed installations, or search path
+        error: Failed to inspect Python interpreter from active virtual environment at `.venv/bin/python3`
+          Caused by: Python interpreter not found at `[VENV]/bin/python3`
```

Comment:
This looks like a bug in our test context; will address that..

Reply:
https://github.com/astral-sh/uv/pull/10716/commits/677e4730a55b79abf6059f8de647257ece4bedb5